src/tests/ftest/server/replay.yaml

```diff
@@ -0,0 +1,36 @@
+hosts:
+  test_servers: 2
+  test_clients: 2
+
+timeout: 130
+
+server_config:
+  name: daos_server
+  engines_per_host: 1
+  engines:
+    0:
+      storage: auto
+
+pool:
+  size: 50%
+  control_method: dmg
+
+container:
+  control_method: daos
+
+ior: &ior_base
+  client_processes:
+    ppn: 4
+  api: DFS
+  transfer_size: 512K
+  block_size: 1G
+  dfs_oclass: SX
+  dfs_dir_oclass: SX
```

Comment:
FYI - IOR doesn't use dfs_dir_oclass. But we specify it all over
```suggestion
```

Reply:
Fixed